### DISCUSSION TITLE
[Feature Request] Choose whether dates outside of the month are disabled or not

### DISCUSSION THREAD

**Discussion by cini9** [heroui-inc/heroui#4710] [Feature requests]
### Is your feature request related to a problem? Please describe.

The feature is not directly related to a problem, but using the Calendar component, it would be great if we could specify whether we want to disable dates that are outside of the month

### Describe the solution you'd like

As a user, I expect to be able to (via props or otherwise) choose whether I want the dates outside of the month to be disabled or not. 

### Describe alternatives you've considered

I've considered using another library like react-calendar, but it would be great if we could have a little bit more customisation options on the Calendar component

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1812/[feature-request]-your-feature-title-here-replace-me">ENG-1812 [Feature Request] YOUR_FEATURE_TITLE_HERE_REPLACE_ME</a></p>
